CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## PENDING
+
+### Features
+- Adding the ability to use refresh tokens with single page applications (SPA) (Early Access feature - reach out to our support team)
+  - `scopes` configuration option now handles 'offline_access' as an option, which will use refresh tokens IF the app of the clientId being used is configured to do so in the Okta settings
```

Comment:
nit: "if the app of the clientId being used" sounds a little off. Can we just tell "If your client app is configured"? Unless we specifically want to mention `clientId` I guess...

Reply:
I was trying to make sure they understood the configuration wasn't in their code (where the `scopes` value under discussion is).  Agree it is awkward though - open to anything that makes that distinction clear.